user_util/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Top-level package for Open edX User Utilities."""
 
-__version__ = '1.0.0'
+__version__ = '1.0.1'
```

Comment:
We should do at least a minor version bump.
```suggestion
__version__ = '1.1.0'
```